### PR TITLE
Release 3.24.0

### PR DESCRIPTION
**Pull request by jezdez**
### Description

✂️ snip snip ✂️ the making of a new release.

Xref #4795

### CODE REVIEW

**Comment by johnnynunez** [comment]
@jezdez new release coming today?

**Comment by jezdez** [comment]
@johnnynunez Possibly today, @kenodegard knows probably best when. That said, it'll take a bit to land in the defaults and conda-forge channels.

**Comment by johnnynunez** [comment]
> @johnnynunez Possibly today, @kenodegard knows probably best when. That said, it'll take a bit to land in the defaults and conda-forge channels.

thanks a lot!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,51 @@
...
+
+* Updated broken links to example conda recipes and updated link to the now archived conda-recipes, with additional links to AnacondaRecipes aggregated feedstocks and conda-forge feedstocks. (#4580)
+* Replaced two instances of "Anaconda Cloud" with "anaconda.org". (#4719)
+
+### Other
+
+* Update test matrix to run tests on all supported Python versions on Linux. Only run tests on lower & upper Python bounds for Windows and macOS. (#4691)
+* Re-enable code coverage reporting to codecov. (#4767)
+* Eliminate test setup's manual clone of https://github.com/conda/conda_build_test_recipe in favor of a session fixture. (#4781)
+* Use tomllib (Python 3.11+) or tomli for .toml support. (#4783)
```
Comment (beeankha) [inline CHANGELOG.md:27]:
```suggestion
* Fix the failing `git clone` when source has LFS files. (#4318)
* Fix many false-positives during the detection of Perl core modules in `conda skeleton cpan`. (#4592)
* `conda skeleton cpan` now correctly adds a C compiler as a dependency if the distribution contains an `.xs` file. (#4599)

### Deprecations

* Removed conda <4.13 logic. (#4677)
* `conda_build.conda_interface.CrossPlatformStLink` is pending deprecation in favor of using `os.stat().st_nlink`. (#4728)
* Drop Python 3.7 support. (#4796)

### Docs

* Updated broken links to example conda recipes and updated link to the now archived conda-recipes, with additional links to AnacondaRecipes aggregated feedstocks and conda-forge feedstocks. (#4580)
* Replaced two instances of "Anaconda Cloud" with "anaconda.org". (#4719)

### Other

* Update test matrix to run tests on all supported Python versions on Linux. Only run tests on lower & upper Python bounds for Windows and macOS. (#4691)
* Re-enable code coverage reporting to `codecov`. (#4767)
* Eliminate test setup's manual clone of https://github.com/conda/conda_build_test_recipe in favor of a session fixture. (#4781)
* Use `tomllib` (Python 3.11+) or `tomli` for `.toml` support. (#4783)
```

**Pull request merged**